### DISCUSSION TITLE
Adjustable Hose/Cable Curves

### DISCUSSION THREAD

**Discussion by Britanniczilla** [tsunamayo/Starship-EVO#5206] [Suggestions]
It, in theory as I haven't seen the games code, would be a simple thing to ad which would also allow builders to do more with the hose/cable system for both decorative and functional purposes. 

The function would work by adding a visual node to the spline when in wiring mode, which could be dragged away and towards the hose connection point to adjust the curve the mesh follows. I've been able to make such a tool in an afternoon with unity, but as I said, I've no idea what the code for this game looks like, only that its made in unity.

A quick image to try and illustrate what I mean: 
https://cdn.discordapp.com/attachments/271723356795961345/1135370509870182513/image.png
The red dot is the control node *assuming its a Bézier curve, judging by how the different hose sizes seem to have control nodes further from the initial node.* 
The red arrow indicates the linier adjustment relative to the base block. These would be visible in the wiring mode.

**Comment by ProPeach** [comment]
This would be incredible if implemented, hoses are one of the best tools to add that "natural" or real feel to builds. If we had just this extra degree of control it would make an already revolutionary brick even better
